File: scripts/update_facility.py
import sys
from facilities import update_facility

facility_name = sys.argv[1]
address = sys.argv[2]
city = sys.argv[3]
state = sys.argv[4]
zip = sys.argv[5]
poc = sys.argv[6]
facility_number = sys.argv[7]
user_id = sys.argv[8]

update_facility(facility_name, address, city, state, zip, poc, facility_number, user_id)
print "done"

File: update_facility.php
<?php

    $facility_name = $_POST["facility_name"];
	$address = $_POST["address"];
	$city = $_POST["city"];
	$state = $_POST["state"];
	$zip = $_POST["zip"];
	$poc = $_POST["poc"];
	$facility_number = $_POST["facility_number"];

	$facility = "'".$facility_name.','.$address.','.$city.','.$state.','.$zip.','.$poc.','.$facility_number."'";
	$execStr = 'python /var/www/html/reunite/scripts/update_facility.py '.$facility;
	$result = exec($execStr);
	//echo $execStr;
	//echo $result;
	//echo "created facility: ".$facility_name;
	header("Location: display_facilities.php");
?>